src/views/Pipeline.vue

```diff
@@ -384,18 +384,27 @@ export default defineComponent({
         this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
           if(event) event.target.complete();
           this.isRetrying = false;
+          emitter.off("selectedShop",this.refreshJobs);
+        }).catch(()=>{
```

Comment:
Why we used catch here?

Reply:
Sir, When the promise is resolved It goes into the 'then' block. If 'then' returns a rejected promise, it goes to catch block. That's why I use the catch here.